server/apps/memory-app/sample-configuration/logback.xml

```diff
@@ -52,29 +43,16 @@
                         <maxFileSize>100MB</maxFileSize>
                 </triggeringPolicy>
 
-                <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
-                        <layout class="ch.qos.logback.contrib.json.classic.JsonLayout">
-                                <timestampFormat>yyyy-MM-dd'T'HH:mm:ss.SSSX</timestampFormat>
-                                <timestampFormatTimezoneId>Etc/UTC</timestampFormatTimezoneId>
-
-                                <!-- Importance for handling multiple lines log -->
-                                <appendLineSeparator>true</appendLineSeparator>
-
-                                <jsonFormatter class="ch.qos.logback.contrib.jackson.JacksonJsonFormatter">
-                                        <prettyPrint>false</prettyPrint>
-                                </jsonFormatter>
-                        </layout>
+                <encoder>
+                        <pattern>%d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
+                        <immediateFlush>false</immediateFlush>
```

Comment:
Why not removing directly the appender `LOG_FILE` as you seem to only get logs from the console now anyways/ :)